src/commands/Minion/bury.ts

```diff
@@ -0,0 +1,105 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { stringMatches, formatDuration, rand } from '../../lib/util';
+import { Time, Activity, Tasks } from '../../lib/constants';
+import { BuryingActivityTaskOptions } from '../../lib/types/minions';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+import Prayer from '../../lib/skilling/skills/prayer';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { SkillsEnum } from '../../lib/skilling/types';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			altProtection: true,
+			oneAtTime: true,
+			cooldown: 1,
+			usage: '<quantity:int{1}|name:...string> [name:...string]',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg: KlasaMessage, [quantity, boneName = '']: [null | number | string, string]) {
+		if (!msg.author.hasMinion) {
+			throw `You dont have a minion`;
+		}
+
+		if (msg.author.minionIsBusy) {
+			return msg.send(msg.author.minionStatus);
+		}
```

Comment:
delete these if's entirely, and replace them with the 2 new decorators, see here for an example: https://github.com/gc/oldschoolbot/blob/bce8983035be343963722be2328f107ab2f8bca1/src/commands/Minion/groupkill.ts#L57

and do this for anywhere else which does these 2 checks.